docs/src/explicit/Extrapolation.md

```diff
@@ -0,0 +1,18 @@
+# OrdinaryDiffEqExtrapolation
+
+This article is a stub.
+
+```@eval
+first_steps = evalfile("./common_first_steps.jl")
+first_steps("OrdinaryDiffEqExtrapolation", "ImplicitEulerExtrapolation")
+```
+
+```@docs
+AitkenNeville
+ImplicitEulerExtrapolation
+ExtrapolationMidpointDeuflhard
+ImplicitDeuflhardExtrapolation
+ExtrapolationMidpointHairerWanner
+ImplicitHairerWannerExtrapolation
+ImplicitEulerBarycentricExtrapolation
+```
```

Comment:
@ChrisRackauckas write a sentence describing what Extrapolation is all about.

All 7 of the solvers are lacking a reference and description of their keywords.

Reply:
Also, putting these solvers under explicit isn't correct, since there are a few implicit ones also.
Would it not make more sense to split this up into ExplicitExtrapolation and ImplicitExtrapolation?

Reply:
https://github.com/SciML/OrdinaryDiffEq.jl/issues/2371#issuecomment-2283654640

Reply:
> Would it not make more sense to split this up into ExplicitExtrapolation and ImplicitExtrapolation?

it would.

Reply:
The extrapolation paper describes it all in too much detail. https://ieeexplore.ieee.org/abstract/document/9926357 . Euler is first order, midpoint and HairerWanner are second order. Deuflhard vs HairerWanner vs Barycentric is somewhat an implementation detail in how the coalescing is performed.